Fails to build with ncurses when libtinfo is separate; needs to use pkg-config

/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.3/../../../../x86_64-pc-linux-gnu/bin/ld: lxdialog/util.o: undefined reference to symbol 'nodelay'
    /usr/lib64/libtinfow.so.6: error adding symbols: DSO missing from command line

See this Gentoo bug for some background info:
https://bugs.gentoo.org/show_bug.cgi?id=457530

crosstool-ng should use pkg-config to figure out what libraries are required for ncurses, instead of blindly looking for a given lib.

--- a/configure.ac
+++ b/configure.ac
@@ -339,6 +339,15 @@ LIBS=$SAVE_LIBS
 AC_SUBST([INTL_LIBS])
 
 #----------------------------------------
+# Check for tinfo, for the kconfig frontends
+AC_SEARCH_LIBS(
+    [tigetnum],
+    [tinfo],
+    [ac_ct_tinfo_lib_found=yes; break])
+AS_IF(
+    [test -z "$ac_ct_tinfo_lib_found"],
+    [AC_MSG_ERROR([could not find tinfo library, required for the kconfig frontends])])
+
 # Check for ncurses, for the kconfig frontends
 AC_SUBST([ac_ct_curses_hdr])
 AC_CHECK_HEADERS(

